app/views/locations/new.erb

```diff
@@ -18,7 +18,14 @@
   <div class='govuk-grid-column-two-thirds'>
     <%= link_to 'Back to list', ips_path, class: 'govuk-back-link' %>
 
-    <h2 class='govuk-heading-l'>Add a new location</h2>
+    <% if current_organisation.locations.empty? %>
```

Comment:
I like the idea, but I worry that it'll be brittle. If we changed the way the user flows through the app, the message might end up confusing.

Still, it's better than what we have now - if no-one else is weighing in with objections, I'd be happy for us to start with this.

Reply:
In case it's confusing in lieu of later comments - this is only a relevant comment if you don't agree with my recommendation to remove this text!